client/static/app.js

```diff
@@ -0,0 +1,86 @@
+const searchbutton = document.querySelector('#searchbutton');
+const form = document.querySelector('form');
+const searchbar = document.querySelector('#searchbar');
+const lucky = document.getElementById('lucky');
+const resultsArea = document.querySelector('#results');
+// const randLink = document.getElementById('randLink');
+
+async function pageLoad(){
+    resultsArea.textContent = "";
+    const url = "http://localhost:4000/results"
+    await fetch(url)
```

Comment:
Usage of await keyword however using .then anyway. You can avoid .then entirely by initialising res in a variable e.g.
`const response = await fetch(url);`
`const data = await response.json();`